yarn-project/aztec-node/src/aztec-node/server.ts

```diff
@@ -784,7 +784,7 @@ export class AztecNodeService implements AztecNode, AztecNodeAdmin, Traceable {
     return new NullifierMembershipWitness(BigInt(index), preimageData as NullifierLeafPreimage, siblingPath);
   }
 
-  async getPublicDataTreeWitness(blockNumber: L2BlockNumber, leafSlot: Fr): Promise<PublicDataWitness | undefined> {
+  async getPublicDataWitness(blockNumber: L2BlockNumber, leafSlot: Fr): Promise<PublicDataWitness | undefined> {
```

Comment:
This inconsistency was triggering me. Unfortunately this is on the public API.

If we would want to not have this change while keeping consistency we would have to rename PublicDataWitness to PublicDataTreeWitness which would result in a monster diff.